README.md

```diff
@@ -107,3 +107,14 @@ Many thanks to the macOS community and special thanks to [mczachurski](https://g
 
 ## License
 [MIT](LICENSE)
+
+## Translation
+
+Equinox is translated to `English` & `French`. To translate Equinox to another language:
+- Fork the main branch 
```

Comment:
Thank you for the awesome guide!

Reply:
😊 There might be another method but this seemed quite a natural way to get the localization working.